docs/reference/commands/setup-passwords.asciidoc

```diff
@@ -3,6 +3,8 @@
 [[setup-passwords]]
 == elasticsearch-setup-passwords
 
+deprecated[7.16,"Replaced by <<reset-elastic-password,`elasticsearch-reset-elastic-password`>>."]
```

Comment:
Do we want to deprecate in 7.16 now?
We haven't backported `elasticsearch-reset-elastic-password` to 7.x yet have we?

It seems premature to indicate that it's deprecated as of 7.16 when we aren't ready to actually deprecate it in 7.x

Reply:
I don't think this is the ideal terminology (though I don't have a better set of words).

There's 3 points:
1. In 8.0 it's "replaced" mostly by security on by default so this utility isn't needed anymore.
2. For some scenarios, `elasticsearch-reset-elastic-password` is a replacement, but not in most
3. For non-elastic users, the replacement is either automated setup (kibana and fleet) or manual use of the change password API

I don't know how we capture that, but I worry that simply pointing to `elasticsearch-reset-elastic-password` will confuse some people.

Reply:
@tvernum, I appreciate your insights, as always. I'd like to avoid version-specific language in this message if possible to avoid pinning to a particular version if we need to change when we're deprecating the tool. Perhaps we can cover the two main scenarios that this tool was used for (changing passwords for built-in users and created users) and offer users the alternative options. 

>The `elasticsearch-setup-passwords` tool is deprecated and will be removed in a future release. To manually reset the password for the `elastic` user, use the 'elasticsearch-reset-elastic-password' tool. To change passwords for other users, use either Kibana or the Elasticsearch change passwords API.

Reply:
Thanks for the suggestions both, I'll add Adam's text here and remove the version